build/yamls/base/antctl.yml

```diff
@@ -50,6 +50,7 @@ rules:
     verbs:
       - get
   - nonResourceURLs:
+      - /loglevel
```

Comment:
I thought these were kept in alphabetical order on purpose?

Reply:
Ok. Changed.